pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1514,7 +1838,12 @@ protected void RunFunctionalTestOrThrow(string defaultUiMode)
 
             UnzipTestFiles();
 
-            _shotManager = new ScreenshotManager(TestContext, SkylineWindow);
+            _shotManager = new ScreenshotManager(SkylineWindow);
+
+            // Run PauseAndContinue ui thread
+            pauseAndContinueThread = new Thread(StartPauseAndContinueForm);
```

Comment:
We need to not create a PauseAndContinueForm until it is first required by a test. Otherwise, the test creates this extra thread and form for all functional tests. In most of our testing, it will never be used.